presto-raptor/src/main/java/com/facebook/presto/raptor/RaptorTableProperties.java

```diff
@@ -70,6 +72,11 @@ public RaptorTableProperties(TypeManager typeManager)
                         "Keep the table organized using the sort order",
                         null,
                         false))
+                .add(booleanProperty(
+                        TABLE_SUPPORTS_DELTA_DELETE,
+                        "Support delta delete on the table",
+                        storageManagerConfig.isTableSupportsDeltaDelete(),
```

Comment:
We don't need a config to flag the table property. Just make it false.

Reply:
> We don't need a config to flag the table property. Just make it false.

According to the early discussion, we want to have the ability to control the default behavior of all the table newly created, instead of making it a manual operation.

Besides, this feature is a good to have, then maybe we don't need to remove it